lib/rich_text_renderer/block_renderers/table_renderer.rb

```diff
@@ -0,0 +1,10 @@
+require_relative './base_block_renderer'
+
+module RichTextRenderer
+  # table node renderer.
+  class TableRenderer < BaseBlockRenderer
+    def render(node)
+      "<table>#{render_content(node)}</table>"
+    end
```

Comment:
nit: Why is this expressed using `#render` rather than `#render_tag` like the other renderers?

Reply:
@cribbles Good catch. I have changed it to `#render_tag` to be consistent with other renderers.